### ISSUE TITLE
Paladins (444090)

### ISSUE THREAD

**Issue by Rabcor** [ValveSoftware/Proton#1787]
# Compatibility Report
- Name of the game with compatibility issues: Paladins
- Steam AppID of the game: 444090

## System Information
- GPU: 980-Ti
- Driver/LLVM version: 396.54.09
- Kernel version: 4.18 (Manjaro)
- Proton version: 3.16-3 Beta

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 

- Fails to install, gets stuck on UE3Redist's attempt at installing .NET Framework (4.0 I think) with a web installer, therefore fails to execute the hirez launcher and cannot be launched.
- After installing Net Framework, the HiRez Launcher shows 'Servers' as Unknown (common issue, not necessarily breaking anything) and gets stuck on 'Paladins is connecting to the Authentication Service'
- Game fails to launch, reportedly because of EasyAntiCheat (unconfirmed, I never got this far, but this is reportedly why nobody has been able to play it on wine since 2016)

**Comment by NTMan** [comment]
## System Information
- GPU: AMD Vega 56/64
- Driver/LLVM version: Mesa 18.3.0-rc4/7.0.0
- Kernel version: 4.19.0-1
- Link to full system information report as [Gist](https://gist.github.com/NTMan/33cca4906328793206218192820779ec)
- Proton version: 3.16-4 beta



[steam-444090.log](https://github.com/ValveSoftware/Proton/files/2630746/steam-444090.log)


## Symptoms 
The game crashed immediately after start.

## Reproduction
Just launch the game.

**Comment by EmmanuelMess** [comment]
## System Information
- GPU: Gráficos HD Intel® 530
- Driver/LLVM version: 3.0 Mesa 18.0.5
- Kernel version:  4.15.0-42-generic
- Link to full system information report as Gist: [Gist](https://gist.github.com/EmmanuelMess/6f67275c18acb5e265cb612618cff3bf)
- Proton version: 3.16-4 beta

## Symptoms 

Kicked out without apparent reason, as if I had exited the match. Might be the anti-hack tool, or a game malfunction. 

## Reproduction

Enqueue a deathmatch. Change resolution(?). Enter a deathmatch. Select Viktor. Wait.

Edit: it seems I cannot enter the game per se, only menues and champion selection.

**Comment by Tank-Missile** [comment]
Since the latest steam beta, I'm no longer being kicked from matches immediately after joining them. .NET Framework 4 still needs to be installed through different means since it gets stuck when trying to install it from steam. For people using or not using the steam runtime, I think these are the changes that fixed the issue:

- Added gnutls 3 to the Steam Runtime, fixing network connectivity issues in many Steam Play titles

- Fixed an issue with host libssl on newer distributions breaking some titles

**Comment by kuasha420** [comment]
Can confirm, the game is fully functional now. Casual and ranked queue both works. Needs to install dotnet40 or mono complete via proton tricks to avoid canceling redist installs on every launch. Make sure to set windows version to windows 7 via winecfg after installing dotnet4 though.

The sudden reason for the success is probably related to the updated EAC binarys that hi rez shipped with the latest patch. A proton supported EAC maybe? People are reportedly having success with other games (dbfz) by copying over paladins eac binaries to those other games. YMMV

**Comment by ghost** [comment]
A couple months ago I emailed every contact available at Hi Rez studio asking them to contact EAC to enable Wine support, specifically Paladins. I got varying responses back from all of them.

I think they actually did.

![import-01-28-19-03-21-33-397550638](https://user-images.githubusercontent.com/12925873/51880022-a64d4500-236d-11e9-8e0c-0da6abe5fb51.png)

**Comment by Unaccounted4** [comment]
The game was working fine until today. I've started to get crashes when joining/in a match.

I haven't really done any system updates or hardware changes.

**Comment by kisak-valve** [comment]
Hello @Unaccounted4, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

**Comment by kuasha420** [comment]
Game still working fine in my end still. I'm on Manjaro Unstable, LLVM9+Mesa-Git, Linux 4.20.5. (R9 390, amdgpu)

**Comment by Unaccounted4** [comment]
I ran a verification of the game files and it found 5 files which didn't match. Which was odd since I only did the delete the UE3*.exes from Redist, so at most it should have been 2.

In any case, false alarm. My bad!

**Comment by Unaccounted4** [comment]
Ever since this issue started happening, I've been getting occasional crashes. They usually manifest after playing 2 or 3 rounds but this time I managed to get a log from a crash on the first match I attempted to join.

Here's the log: 
[steam-444090.log](https://github.com/ValveSoftware/Proton/files/2823502/steam-444090.log)

Here are my Steam specs:
https://gist.github.com/Unaccounted4/b63fcbd46be2e7746705a3443601b08a

**Comment by Tank-Missile** [comment]
Are you sure you installed .NET Framework 4 properly? The only way I've gotten it to install was by using protontricks, although I did make note that the game started without it. The errors I'm seeing in the log seem to hint at missing dlls in the Proton 3.16 Beta/dist/lib/wine directory, but I don't seem to have those dlls either.

**Comment by Unaccounted4** [comment]
Yes, this happens both with and without dotnet40 installed. I did ponder that not having .NET installed would cause issues so I decided to try with, as well.

Doesn't seem to make a difference, though. I can get the game to crash reliably by playing three matches in a row by just hitting Requeue at the end of each one. Though sometimes, it'll happen on the very first try as well.

It is worth mentioning, with .NET installed, I get the AutoReporter.exe window after the crash. It doesn't appear if I delete the UE3*.exe files instead.

**Comment by Tank-Missile** [comment]
This might be a long shot, but what does winecfg report the windows version to be in the prefix? I'm assuming you're using protontricks. Set it to windows 7 if it's not already, and start the game again. It should then prompt for the installation of two different redistributables. winetricks has workarounds for dotnet40 involving setting the prefix to Windows XP, which interferes with redistributable installation.

**Comment by Unaccounted4** [comment]
I did notice that output from protontricks and immediately ran protontricks win7 the moment it finished installing dotnet40. So, the game never even started in WinXP mode.

I have yet to try completely wiping the game and prefix and reinstalling. I will try that next! Maybe a fresh install will help.

**Comment by Unaccounted4** [comment]
Alright, so today a bunch of things happened: the Solus repos sync'ed and a bunch of updates came down the pipe, I removed and reinstalled the game, and there were two Steam updates. One or a combination of these seems to have "fixed" whatever was going on. I played 6 matches in a row and no crashes to report.

Remains to be seen if this will stay fixed in the near future.

**Comment by kuasha420** [comment]
Paladins is no longer working. Multiplayer match kicks me out to the main menu as soon as the loading screen comes up or when selecting talent. Back to square one on this one.

**Comment by BoldyPlays** [comment]
I can confirm Kuasha420's post.. Behaving like a EAC kick, but I couldn't say for sure.

**Comment by AdsonCicilioti** [comment]
I also Confirm.. Kick to menu. :(

**Comment by iLynux8888** [comment]
Confirmed the game is now crashing. Using Fedora 29, Proton 3.16-6 Beta. I've reinstalled the game and it hasn't helped. Used Winetricks to install dotnet40 and that didn't help. I'm going to change the Wine compatibility to Windows 7 instead of XP (default) and see if it helps.

Edit: That did not help. CRAP!

**Comment by Tank-Missile** [comment]
I'm afraid I have to confirm with everyone else. I too am being kicked from matches after about 15 seconds of entry. EAC needs to be snapped from this universe.

**Comment by iLynux8888** [comment]
I suspect that the Linux functionality was inadvertently broken on Saturday from a server-side perspective, in preparation of the new patch that enabled Crossplay with XBox One and Nintendo Switch. I have no proof of this, but the timing is suspect.

**Comment by sysrq-reisub** [comment]
It's working for me now

**Comment by kuasha420** [comment]
Really? I'll have to fire it up and test. Is it the futures end update?

**Comment by sysrq-reisub** [comment]
Yes, I don't know why EAC doesn't kick me out aynmore

**Comment by BoldyPlays** [comment]
Downloading right now. The futures end and the release of the new champion (Atlas) was yesterday I think. Hope this is a sign of future cooperation between Proton and EAC.

**Comment by kuasha420** [comment]
By the way are you testing on proton 3.16 or 4.2?

**Comment by BoldyPlays** [comment]
4.2 attempted, installed dotnet40 through protontricks. Away she goes.

Looks like ProtonDB has plenty of reports to back it up too.

See you there champions. Need Healing.

**Comment by sysrq-reisub** [comment]
Works fine, but when I'm using the mouse with 1000hz dpi polling rate, the fps drop is very noticeable

**Comment by Tank-Missile** [comment]
Using proton 4.2, but the game is actually even less functional for me. It crashes right after the splash screen. I've installed dotnet40 into the prefix, so this is quite peculiar.

EDIT: Okay, so I figured out what the issue was. Starting from a new prefix, I installed dotnet40 using protontricks, and then set the windows version to 7. This allowed the redistributables to install. However, the game refused to launch due to the windows version being set to 7. To rectify this, I set the windows version back to XP. The game then launched. When queueing for a match, I was not kicked by EAC upon entering the map! I have no idea why the windows version being set to 7 would cause the game to crash, but I guess it's not worth investigating.

**Comment by kuasha420** [comment]
Everything you said, I also had to set win xp otherwise it would crash. 

Also, there's two rundll32.exe error before the game is able to start

**Comment by puntodamar** [comment]
should i use a dummy account first just in case my real account get blocked?

**Comment by KibaGasteiz** [comment]
The game works fine now, but at some point it stutters like hell, dips from 150fps to 20fps. I need to restart the game every match so i can play with decent framerate.
Since github won't let me upload a huge log, here you got a mega link: https://mega.nz/#!fBlTWahD!fwmbsikRljbrQ-fbbDjau3aYH3aUMm_0IW4G3laP-Yc

**Comment by kisak-valve** [comment]
Hello @KibaGasteiz, in general proton logs compress well in an archive if you feel like uploading large logs to Github.

**Comment by KibaGasteiz** [comment]
Yes, but it's a log for an entire match of Paladins, because the game starts stuttering when i've played for some minutes. Maybe it's something related to ram or vram memory? I will check resources now.
EDIT: Every match i got 0.2GB additional RAM used on the menu. I don't know if this is the problem or it's normal.

**Comment by kuasha420** [comment]
@KibaGasteiz what's your gpu driver?

**Comment by KibaGasteiz** [comment]
Intel(R) Core(TM) i5-6600K CPU @ 3.50GHz
Kernel: 5.0.9-arch1-1-ARCH
DE: GNOME Shell
Steam runtime: jenkins-steam-runtime-beta-release_0.20190320.2

NVIDIA Corporation GeForce GTX 1070
**NVIDIA 418.56**

RAM: 15980 MB

**Comment by KibaGasteiz** [comment]
So i've been testing and...
The game stutters with Vsync enabled too, but not like with no vsync (i use to disable that option in every game). I can play a lot of matches without getting 20fps or less, but it's still not perfect since sometimes i got hard stuttering.

**Comment by kuasha420** [comment]
New Paladins Update (probably unintentionally) broke EAC again. (2.6). For fucks sake Hi-Rez...

**Comment by turanmahmudov** [comment]
@kuasha420 happened to me too. Game kicks me out

**Comment by ShaneTursam** [comment]
@kuasha420 @turanmahmudov  I'm also having the same issue, was working fine up to the last update.

**Comment by Julien684** [comment]
@kuasha420 I have the same problems as you, here are the log at the time of the kick of the game :
[log-Paladins-Kick.txt](https://github.com/ValveSoftware/Proton/files/3219577/log-Paladins-Kick.txt)

**Comment by KibaGasteiz** [comment]
Some guy here says that the game works if you use Wine4.8 mixed with Proton: https://www.protondb.com/app/444090
Has Wine4.8 did something about anti-cheats or is he drunk?

**Comment by turanmahmudov** [comment]
> Some guy here says that the game works if you use Wine4.8 mixed with Proton: https://www.protondb.com/app/444090
> Has Wine4.8 did something about anti-cheats or is he drunk?

how to use wine4.8 mixed with proton? any guide?

**Comment by ShaneTursam** [comment]
@turanmahmudov I don’t think you can update the version of wine proton uses. You could download that version of wine and install the windows version of steam and redownload the game plus install the dot40. I don’t know if that would get you going though. I’d say the guy is drunk.

**Comment by kuasha420** [comment]
I've tried 4.8 version of wine. Same issue. Kick to menu after loading into a match.

**Comment by 414n** [comment]
Any changes using latest Proton 5?

**Comment by LouisMarotta** [comment]
> Any changes using latest Proton 5?

still cant play without getting kicked (proton 5.0-3)

**Comment by KhanSW** [comment]
splash screen then kick on latest proton 5.0-4

**Comment by AnasR7** [comment]
> splash screen then kick on latest proton 5.0-4

yes i have same issues.. with proton 5.0.5

**Comment by 414n** [comment]
I've had some success in DBFZ: https://github.com/ValveSoftware/Proton/issues/1240#issuecomment-625477935.
Maybe you can try applying similar instructions for Paladins too?

**Comment by arafatzahan** [comment]
Now that Denuvo Anti Cheat is supporting Proton, EAC should take note. 
https://github.com/ValveSoftware/Proton/issues/3773#issuecomment-631750507

**Comment by ryao** [comment]
@aeikum This patch in proton broke paladins:

https://github.com/ValveSoftware/wine/commit/f48a97689fe92d2831cec13e7628640d33cd2989

Reverting the patch fixes paladins, but causes Sword Art Online: Fatal Bullet to hang at start.

**Comment by utherbone** [comment]
Hi, the game runs very well I risk saying it as well as in the windows, however the game kick me as soon as the game starts what could it be? EAC?

**Comment by kuasha420** [comment]
Has anything changed regarding this game, EAC and Proton?

**Comment by soldtm** [comment]
Works fine on Ubuntu 21.04 & Proton 4.11-13 with dotnet40 and winxp
`protontricks 444090 dotnet40 winxp`

UPD: works from time to time 
UPD 21.10.21: seems they enabled Wine support, game works great for me on Proton-6.18-GE-2

**Comment by jaw-sh** [comment]
Cannot get game to run beyond splash screen using any Proton library. Earlier libraries cannot get splash screen. Later ones will close silently after splash.

5.12.19-1-MANJARO
Intel© Core™ i7-7700K CPU @ 4.20GHz × 4
NVIDIA Corporation GP104 [GeForce GTX 1080]

**Comment by soldtm** [comment]
> Cannot get game to run beyond splash screen using any Proton library. Earlier libraries cannot get splash screen. Later ones will close silently after splash.
> 
> 5.12.19-1-MANJARO Intel© Core™ i7-7700K CPU @ 4.20GHz × 4 NVIDIA Corporation GP104 [GeForce GTX 1080]

try another Proton version, i'm using Proton-6.18-GE-2 (https://github.com/GloriousEggroll/proton-ge-custom), game works great

**Comment by Newbytee** [comment]
> UPD: works from time to time

Could you elaborate on what you mean by this exactly?

**Comment by elch01** [comment]
Installing works just fine and booting requires Proton-6.21-GE-1.
But it seems like i get kicked form every match so i assume its EAC related, loader.log seems to imply it isn't downloading the latest EAC client at all thus anti-cheat is non functional.
Is this something that can be fixed locally or does Hi-Rez need to do something?

**Comment by Newbytee** [comment]
> Installing works just fine and booting requires Proton-6.21-GE-1. But it seems like i get kicked form every match so i assume its EAC related, loader.log seems to imply it isn't downloading the latest EAC client at all thus anti-cheat is non functional. Is this something that can be fixed locally or does Hi-Rez need to do something?

Hi-Rez said that they are looking into enabling Proton support for EAC in Paladins, but that it likely won't happen in December (when the Deck originally was meant to launch): https://www.theverge.com/2021/10/5/22709918/valve-steam-deck-supported-games-anti-cheat-proton-eac-battleye-epic

So no, until that happens there's nothing you realistically can do.

**Comment by soldtm** [comment]
> > UPD: works from time to time
> 
> Could you elaborate on what you mean by this exactly?

i meant game sometimes starts but sometimes not

**Comment by tridoxx** [comment]
hello, the game seems to work correctly, but after 30 seconds of starting, it takes us to the main menu, I was looking at the steam logs and it throws this problem. 

0025.12] Log: *** WARNING - PATHS MAY NOT BE VALID ***

[0025.12] Log: Skipping initialization of FSMPartPool!
SwfMovie MenuManager.undefined

**Comment by FirephoenixX02** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/1787#issuecomment-968325043

They haven't enabled it yet. Hope they enable it in the next month or so and not next year.
But besides EAC the game works great with Proton-7.1-GE-2. I get around 120-140 FPS with a 1050ti. I set Texture Detail to very high everything else to Medium. I also didnt have to install dotnet40 or anything like that.

**Comment by dolphinau** [comment]
> > Replying to [#1787 (comment)](https://github.com/ValveSoftware/Proton/issues/1787#issuecomment-968325043)
> 
> They haven't enabled it yet. Hope they enable it in the next month or so and not next year. But besides EAC the game works great with Proton-7.1-GE-2. I get around 120-140 FPS with a 1050ti. I set Texture Detail to very high everything else to Medium. I also didnt have to install dotnet40 or anything like that.

Does the game work for you? I mean no kicking after 3sec in a battle? I still have this bug with Proton-7.1-GE-2

**Comment by FirephoenixX02** [comment]
> Does the game work for you? I mean no kicking after 3sec in a battle? I still have this bug with Proton-7.1-GE-2

That kicking after 3sec in batlle is not a Proton Bug it's the problem with EAC, as already said.

**Comment by dolphinau** [comment]
> > Does the game work for you? I mean no kicking after 3sec in a battle? I still have this bug with Proton-7.1-GE-2
> 
> That kicking after 3sec in batlle is not a Proton Bug it's the problem with EAC, as already said.

Yep of course, but the EAC issue hasn't been patched yet, so currently the game doesn't work

**Comment by FirephoenixX02** [comment]
> Yep of course, but the EAC issue hasn't been patched yet, so currently the game doesn't work

Depends on how you define working. If you define ist that all Game mechanics are working with good Performance then its working. If you define it that you can Play like anyone on Windows without Limitations then its Not working.

**Comment by dolphinau** [comment]
> Depends on how you define working. If you define ist that all Game mechanics are working with good Performance then its working. If you define it that you can Play like anyone on Windows without Limitations then its Not working.

I define "working" by I can play a game! Thus I'll assume it doesn't work for the moment

**Comment by ahcm** [comment]
If the problem is that the Paladins developers do not enable playing via Proton in their EAC configuration, this is not a Proton issue. Therefore you need to take this to the Paladins developers (Hi-Rez).

**Comment by FirephoenixX02** [comment]
@atxr @tridoxx @Rabcor @everyoneelse
As of now the Game is working with Proton-GE-7.1 !!! (including Multiplayer) !!!.
But be aware that by playing on Linux your Account can be banned due to Hi-Rez not officially supporting Linux / Proton!
Also this might be only temporary so keep that in mind.
Edit: It's broke again, for me at least. Looks like it was working for 5 days until now.

**Comment by kuasha420** [comment]
@FirephoenixX02 You mean `Proton-GE-7.1` or `Proton-GE-7.18` (Latest) ?

**Comment by FirephoenixX02** [comment]
> @FirephoenixX02 You mean `Proton-GE-7.1` or `Proton-GE-7.18` (Latest) ?

I mean 7.1, i tried it with 7.18 but it immediately close after start.
Edit: Button Problem is fixed after complete restart. Game is working again.

**Comment by GunpowderGuy** [comment]
The game no longer works with ge proton 7-20

**Comment by FirephoenixX02** [comment]
> The game no longer works with ge proton 7-20

It never worked with any Versions after GE 7.1 for me. Should work with 7.1 still but it could have changed in an Update.

**Comment by GunpowderGuy** [comment]
I have checked with several versions and the only one that works for me is 6-21-GE-2. Has hi rez announced whether they will officially support proton? another of their games recently became steam deck certified

**Comment by FirephoenixX02** [comment]
> I have checked with several versions and the only one that works for me is 6-21-GE-2. Has hi rez announced whether they will officially support proton? another of their games recently became steam deck certified 

Officially Paladins DOES NOT support Proton/Wine it is likely that they just disabled the Anticheat.

**Comment by Friz64** [comment]
There was a fix in Proton Experimental for getting the game to run: https://github.com/ValveSoftware/Proton/wiki/Changelog#available-in-proton-experimental-as-of-20220622

Here's a video of it running on the Steam Deck: https://youtube.com/watch?v=pHl_98zBG3s

Edited for clarity.

**Comment by FirephoenixX02** [comment]
> As noted in the latest Proton Experimental Release Notes, Paladins is now officially supported with Proton Experimental: https://github.com/ValveSoftware/Proton/wiki/Changelog#available-in-proton-experimental-as-of-20220622
> 
> Here's a video of it running on the Steam Deck: https://youtube.com/watch?v=pHl_98zBG3s

So Proton supports Paladins but does Hi-Rez support Proton? Because Paladins previously also ran fine with Proton, but after an update the Anticheat was enabled again and was broken again. I just don't wan't to download 30GB and delete it in 2-3 Weeks again.

**Comment by GunpowderGuy** [comment]
-Did anyone else notice better graphics with proton experimental than with that old proton ge version?
-Is anyone else sometimes having trouble starting the game until they reset their computer?

**Comment by FirephoenixX02** [comment]
> -Did anyone else notice better graphics with proton experimental than with that old proton ge version?
> -Is anyone else sometimes having trouble starting the game until they reset their computer?

- It never worked for me with Proton Experimental in the first place so i can't answer that
- No i never had any trouble opening it, atleast not when using Proton GE, Experimental doesn't work for me as stated  above, so might be a problem with Proton Experimental if you use that to run the game, but don't quote me on that could also be something different.

**Comment by Chocrates** [comment]
Working mostly fine with Proton Experimental.  It has been crashing mid-game a few times.  
Attached is the log if it proves useful.
[steam-444090.log](https://github.com/ValveSoftware/Proton/files/9065936/steam-444090.log)

Proton-GE-7.24 seemed to work better (less crashes) but alt-tabbing was not available.  Alt Tabbing is working with Proton Experimental.

**Comment by tbsteinb** [comment]
I'm not having any luck with it on proton experimental.

Looking at loader.log there seems to be an improvment. In the past it would go to download some files for EAC, get a 403 and fail. Now it gets a 200, spends a while failing to download the file and then EAC gives a callback code of 915 (same one it gave in the past). 

It tries to download from https://download.eac-cdn.com/api/v1/games/112/client/wine64/download/?uuid=some-uuid

Going to the same url in my browser downloads a 0 byte file. Any ideas?

**Comment by kisak-valve** [comment]
EAC appears to be broken on EAC and Paladins again

Issue transferred from https://github.com/ValveSoftware/Proton/issues/5984.
@AdverseMiller posted on 2022-07-11T20:34:03:

As of writing this, EAC is broken doesn't work anymore on either of these games.

**Comment by tbsteinb** [comment]
> EAC appears to be broken on EAC and Paladins again
> 
> Issue transferred from #5984. @AdverseMiller posted on 2022-07-11T20:34:03:
> 
> As of writing this, EAC is broken doesn't work anymore on either of these games.

It seems a bit different from past breakages. Just in that the EAC link returns a 200 status instead of a 403 status. Though I might be going off old data. Was it giving a 200 status last time Paladins worked?

**Comment by waspennator** [comment]
Think someone might need to change the "playable" Deck rating back to unsupported if EAC is booting people off until this gets fixed, cause I'm seeing alot of reports on protondb bringing this up.

**Comment by sekwah41** [comment]
@waspennator I literally tried it now and it did that so it seems to still be the case

**Comment by LepaBrenaa** [comment]
It was stated numerous times on reddit, Paladins will always work for few days after an update. After those few days EAC starts working as intended, and people get kicked out of matches therefore.
As long as Paladins doesn't enable EAC support, it will only work for few days after any update.

**Comment by GunpowderGuy** [comment]
Is there a workaround to enable eac linux support with paladins?
Does this game run under virtualization?

**Comment by LepaBrenaa** [comment]
There is no workaround. It will most likely work again out of the box for few days after the patch comes out with the new champion.
After that EAC will work properly again, and people will be kicked out of matches.

**Comment by kuasha420** [comment]
The Full RWBY patch was playable though.

**Comment by LepaBrenaa** [comment]
What do you mean by "Full"?
The patch came, and the game was playable for few days. After few days everyone started to get kicked out of games due to EAC.
Not a single regular patch has been released since then.
The next patch will be the one with new champion.

**Comment by sekwah41** [comment]
Seems the issues with paladins isn't from the fact wine or the anti cheat don't support each other, but rather the developer just can't be asked to change it.

There are plenty of games that use the same anticheat that don't kick or break with Linux users.

Someone feel free to correct me on this but they released support [about a year ago now](https://dev.epicgames.com/en-US/news/epic-online-services-launches-anti-cheat-support-for-linux-mac-and-steam-deck?sessionInvalidated=true).

So it seems either the devs are not updating it or cant be asked to. Whatever the reason unless epic is being misleading it should be as easy as a few clicks.

"developers can activate anti-cheat support for Linux via Wine or Proton with just a few clicks in the Epic Online Services Developer Portal"

Though seeing as some of the Linux files for EAC seem missing it could be an update that is stopping them.

**Comment by waspennator** [comment]
Still wonder if Paladins will get bumped back down to unsupported on deck soon, cause the latest version of Proton just dropped and it's still marked as playable for some reason.

**Comment by Kreyren** [comment]
Cross-referencing Valve+Hirez fixing EAC for SMITE that might enable fixing paladins as well: https://github.com/ValveSoftware/Proton/issues/2208#issuecomment-1565082515

**Comment by massatt212** [comment]
i use to play paladins every day after work, since i swap to linux i cant enjoy it anymore, i tried overwatch and it doesnt feel the same, i believe Paladins is much better than.

**Comment by kisak-valve** [comment]
[EAC][DT_HASH] Paladins EAC always kick in match

Issue transferred from https://github.com/ValveSoftware/Proton/issues/7259.
@EliasTheGrandMasterOfMistakes posted on 2023-11-14T16:38:49:

# Compatibility Report
- Name of the game with compatibility issues: Paladins
- Steam AppID of the game:444090

## System Information
- Proton version: Proton Experimental

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.



## Symptoms 
Because GNU LIBC (GLIBC) have dropped DT_HASH a some years ago(Thanks Glibc high stable ABI, thanks thanks thanks), have more than 2 years that Paladins can't work because anticheat kicks during match, for some whatever reason trying build GLIBC with DT_HASH patch not working, idk if is necessary for elftools have be patch for work.
Have a way to Proton can make this work? a specific workaround for Anticheat or some thing similar?
Have a way to contact Hi-rex ?
Hi Rex don't look to fix this and bring this for linux (conspiracy theory time) (aluminum hat is ready to use)

**Comment by kisak-valve** [comment]
Paladins (444090)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/9351.
@yobenbobin posted on 2025-12-30T20:25:22:

# Compatibility Report
- Name game:Paladins
- AppID:444090

## System Information
- GPU:NVIDIA Corporation TU117M [GeForce MX450]
- Video driver version: 4.6.0 NVIDIA 580.95.05
- Kernel version: 6.14.0-37-generic
- Link to full system information: https://gist.github.com/yobenbobin/869d6e5af494b18a519fb7645be37305
- Proton version: proton-10.0-4-rc1

## I confirm:
- [ ✔] that I haven't found an existing compatibility report for this game.
- [ ✔] that I have checked whether there are updates for my system available.

## Symptoms 
When “use dx11” is selected in the settings, meaning the game runs on DirectX 11, then when you press Alt+Tab, Super+D, or simply go to the desktop, you get a black screen and there's nothing you can do about it. But if you disable the setting, meaning run the game on DirectX 9, then everything is fine. I installed a custom proton and the situation is the same.

- This issue is specific to **Fullscreen mode**. 
- In **Borderless Window** mode, Alt+Tabbing works as expected without any black screen.

## Reproduction

[steam-444090.log](https://github.com/user-attachments/files/24387509/steam-444090.log)

1. Launch Paladins via Proton.
2. Open **Settings** -> **Video** tab.
3. Enable **"Use DX11"** and set **Display Mode** to **"Fullscreen"**.
4. Restart the game to apply changes (if necessary).
5. Once in the main menu or in a match, press **Alt+Tab** or the **Super key** (Windows key) to switch to the desktop.
6. Observe that the screen turns black and the system/game becomes unresponsive.
7. Note: The issue does not occur if "Borderless Window" is used instead of "Fullscreen".
